src/background.js

```diff
@@ -1,29 +1,44 @@
 window.scrNoti = window.scrNoti || {};
+const seenMessages = {};
+
+// TODO:
```

Comment:
For the final merge, please remove those "TODO" from the code and create issues if you want for them.

Reply:
That was my plan anyway, but good to have a reminder here!

Reply:
Done.